integreat_cms/cms/models/statistics/page_accesses.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from django.core.validators import MinValueValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from ..abstract_base_model import AbstractBaseModel
+
+
+class PageAccesses(AbstractBaseModel):
+    """
+    Data Model representing the accesses of a page
+    """
+
+    access_date = models.DateTimeField(
+        verbose_name="Date of the page accesses",
+    )
+    language = models.ForeignKey(
+        "cms.Language",
+        on_delete=models.CASCADE,
+        verbose_name="Languages of the page that was accessed",
+    )
+    page = models.ForeignKey(
+        "cms.Page",
+        on_delete=models.CASCADE,
+        verbose_name="Accessed page",
+    )
+    accesses = models.IntegerField(
+        validators=[MinValueValidator(0)], verbose_name="Page accesses"
+    )
+
```

Comment:
At the moment there is no __str__() method. Therefore the objects of this model appear as "PageAccesses object (1)" in the Django Admin. I think it might be helpful to have a __str__() method to have a more speaking name. This could look something like this 

```suggestion
def __str__(self):
        return f"{self.page} - Accesses: {self.accesses}, language: {self.language}, date: {self.access_date}"
```